cmd/client/packet_injector.go

```diff
@@ -198,6 +200,7 @@ func addInjectPacketFlags(cmd *cobra.Command) {
 	cmd.Flags().Int64VarP(&incrementPayload, "incrementPayload", "", 0, "increase payload for each packet")
 	cmd.Flags().Int64VarP(&count, "count", "", 1, "number of packets to be generated")
 	cmd.Flags().Int64VarP(&interval, "interval", "", 1000, "wait interval milliseconds between sending each packet")
+	cmd.Flags().Int64VarP(&ttl, "ttl", "", 0, "ttl in seconds")
```

Comment:
TTL is not in second, it is the number of Hop the packet is allowed to pass through.

Reply:
oops :-) modified.